tests/CommandsTest.php

```diff
@@ -179,6 +179,17 @@
         ->expectsOutput('Tenant: ' . $tenantId2);
 });
 
+test('run command works when sub command asks question and accepts argument', function () {
+    $id = Tenant::create()->getTenantKey();
+
+    Artisan::call('tenants:migrate', ['--tenants' => [$id]]);
```

Comment:
What's this here for?

Reply:
Ah, I was migrating tenant migrations because I was dumping tenant users in the `Run.php` command for testing purposes. So this is a leftover code. I'll remove it, and tests will pass without this line.